pyked/converters.py

```diff
@@ -0,0 +1,522 @@
+
+    # now get data points
+    datapoints = []
+    for dp in dataGroup.findall('dataPoint'):
+        datapoint = {}
+        for val in dp:
+            units = unit_id[val.tag]
+            if units == 'Torr':
+                units = 'torr'
+            datapoint[property_id[val.tag].replace(' ', '-')] = [val.text + ' ' + units]
+        datapoints.append(datapoint)
+
+    if len(datapoints) == 0:
+        raise MissingElementError('dataPoint')
+
+    # RCM files may have a second dataGroup with volume-time history
+    if len(dataGroups) == 2:
+        dataGroup = dataGroups[1]
+        time_tag = None
+        volume_tag = None
+        for prop in dataGroup.findall('property'):
+            if prop.attrib['name'] == 'time':
+                time_dict = {'units': prop.attrib['units'], 'column': 0}
+                time_tag = prop.attrib['id']
+            elif prop.attrib['name'] == 'volume':
+                volume_dict = {'units': prop.attrib['units'], 'column': 1}
+                volume_tag = prop.attrib['id']
+            else:
+                raise KeywordError('Only volume and time allowed in volume history dataGroup.')
+
+        if time_tag is None or volume_tag is None:
+            raise KeywordError('Both time and volume properties required for volume history.')
+        volume_history = {'time': time_dict, 'volume': volume_dict, 'values': []}
+
+        # collect volume-time history
+        for dp in dataGroup.findall('dataPoint'):
+            time = None
+            volume = None
+            for val in dp:
+                if val.tag == time_tag:
+                    time = float(val.text)
+                elif val.tag == volume_tag:
+                    volume = float(val.text)
+                else:
+                    raise KeywordError('Only volume and time values allowed in '
+                                       'volume-history dataPoint.'
+                                       )
+            if time is None or volume is None:
+                raise KeywordError('Both time and volume values required in each '
+                                   'volume-history dataPoint.'
+                                   )
+            volume_history['values'].append([time, volume])
+
+        datapoints[0]['volume-history'] = volume_history
+
+    elif len(dataGroups) > 2:
+        raise NotImplementedError('More than two DataGroups not supported.')
+
+    return datapoints
+
+
+def ReSpecTh_to_ChemKED(filename_xml, filename_ck='', file_author='', file_author_orcid=''):
+    """Convert ReSpecTh XML file to ChemKED YAML file.
+
+    Args:
+        filename_xml (`str`): Name of ReSpecTh XML file to be converted.
+        filename_ck (`str`, optional): Name of output ChemKED file to be produced.
+        file_author (`str`, optional): Name to override original file author
+        file_author_orcid (`str`, optional): ORCID of file author
+    """
+    from pyked.chemked import ChemKED
```

Comment:
You can do `from . import chemked` and then below use `chemked.ChemKED(...`. That doesn't raise any circular import errors, I don't think